### ISSUE TITLE
typing 'git pull' gave me an error in terminal

### ISSUE THREAD

**Issue by miclp** [learn-co-curriculum/git-remotes-with-github-readme#6]
At the end of the lesson, I tried 'git pull' in terminal and received this error:

fatal: No remote repository specified.  Please, specify either a URL or a
remote name from which new revisions should be fetched.

It only worked when I typed:
'git pull destination master'

MP

**Comment by AnnJohn** [comment]
hi @miclp did 'git pull origin master' work? we'll need to update this but wanted to check in on this. thanks!

**Comment by miclp** [comment]
Hi @AnnJohn, thanks for following up.  The reason I typed 'git pull destination master' is actually related to the pull request I opened on this same lesson.  The reason being:  the walkthrough previously had me rename 'origin' to 'destination' (on line 35) so I believe I received an error if I typed 'git pull origin master' , whereas 'git pull destination master' did work.  For this reason, all the commands after line 35 needed the word 'destination' in place of 'origin'.  Hope this helps.. I'm new to git but I'm pretty sure I have this right.  -

-An alternate solution would be to omit line 35 and NOT have students rename the remote repo. (and keep the commands afterward with the word 'origin').
